test/e2e/service.go

```diff
@@ -1337,39 +1337,57 @@ var _ = framework.KubeDescribe("Services", func() {
 		framework.CheckReachabilityFromPod(true, normalReachabilityTimeout, namespace, dropPodName, svcIP)
 	})
 
-	It("should be able to create an internal type load balancer on Azure [Slow]", func() {
-		framework.SkipUnlessProviderIs("azure")
+	It("should be able to create an internal type load balancer [Slow]", func() {
```

Comment:
I moved the ILB-enabling/disabling portion of the test to framework. The only cloud-specific portion left is the internal static IP for azure which I believe will be removed in a few weeks after adding support for GCE.